nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -355,7 +355,7 @@ int ddxProcessArgument(int argc, char *argv[], int i)
       /* FIXME: why limit 10 1024 bytes? */
       if ((size = strlen(argv[i])) < 1024)
       {
-        if ((nxagentOptionsFilename = strndup(argv[i], size)) == NULL;
+        if ((nxagentOptionsFilename = strndup(argv[i], size)) == NULL);
```

Comment:
the semicolon at the line end feels wrong. Also: please rebase with the previous commit that you try to fix with this commit.